krux_boto/cli.py

```diff
@@ -31,6 +32,8 @@ def __init__(self, name=NAME):
 
         self.boto = get_boto(self.args, self.logger, self.stats)
 
+        self.boto3 = get_boto3(self.args, self.logger, self.stats)
+
```

Comment:
the bootstrap is straightforward - that's just a python object (no network traffic).

This gives you the flexibility to use either, and as they're not just api incompatible, but also conceptually incompatible, this lets us do a piecemeal migration.

And on top of that, a bunch of new features will only be in boto3.

So yeah, awesome.